x-pack/legacy/plugins/index_lifecycle_management/public/sections/edit_policy/components/min_age_input.js

```diff
@@ -88,6 +93,39 @@ export const MinAgeInput = props => {
         defaultMessage: 'hours from rollover',
       }
     );
+    minutesOptionLabel = i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.rolloverMinutesOptionLabel',
+      {
+        defaultMessage: 'minutes from rollover',
+      }
+    );
+
+    secondsOptionLabel = i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.rolloverSecondsOptionLabel',
+      {
+        defaultMessage: 'seconds from rollover',
+      }
+    );
+    millisecondsOptionLabel = i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.rolloverMilliSecondsOptionLabel',
+      {
+        defaultMessage: 'milliseconds from rollover',
+      }
+    );
+
+    microsecondsOptionLabel = i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.rolloverMicroSecondsOptionLabel',
+      {
+        defaultMessage: 'microseconds from rollover',
+      }
+    );
+
+    nanosecondsOptionLabel = i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.rolloverNanoSecondsOptionLabel',
+      {
+        defaultMessage: 'nanoseconds from rollover',
+      }
+    );
   } else {
```

Comment:
I think this `else` statement will need to be updated as well. I currently see this in the UI when I disable rollover.

<img width="352" alt="Screen Shot 2020-01-17 at 1 40 35 PM" src="https://user-images.githubusercontent.com/5226211/72637410-401b5280-392f-11ea-8757-ccba1db74edd.png">